test/wasmBenchmarker/benchmark.py

```diff
@@ -161,13 +164,13 @@ def run_wasm(engine, path, name):
     return True
 
 
-def measure_time(path, name, functon, engine=None):
+def measure_time(path, name, functon, engine=None, jit=False):
```

Comment:
This is not introduced here, but `functon` should be changed to `function`. Otherwise the patch ok.